feedbacks.js

```diff
@@ -0,0 +1,286 @@
+					choices : self.auxTimelinesChoices,
+					default : ''
+				},
+				{
+					type: 'checkbox',
+					label: 'Use icon instead of colors',
+					id: 'icons',
+					default: true
+				},
+				{	// Play FG
+					type    : 'colorpicker',
+					label   : 'Play foreground color',
+					id      : 'playfg',
+					default : self.feedbacksSettings.colors.task.play.fg
+
+				},
+				{	// Play BG
+					type    : 'colorpicker',
+					label   : 'Play background color',
+					id      : 'playbg',
+					default : self.feedbacksSettings.colors.task.play.bg
+				},
+				{	// Pause FG
+					type    : 'colorpicker',
+					label   : 'Pause foreground color',
+					id      : 'pausefg',
+					default : self.feedbacksSettings.colors.task.pause.fg
+
+				},
+				{	// Pause BG
+					type    : 'colorpicker',
+					label   : 'Pause background color',
+					id      : 'pausebg',
+					default : self.feedbacksSettings.colors.task.pause.bg
+				},
+				{	// Stop FG
+					type    : 'colorpicker',
+					label   : 'Stop foreground color',
+					id      : 'stopfg',
+					default : self.feedbacksSettings.colors.task.stop.fg
+
+				},
+				{	// Stop BG
+					type    : 'colorpicker',
+					label   : 'Stop background color',
+					id      : 'stopbg',
+					default : self.feedbacksSettings.colors.task.stop.bg
+				}
+			],
+			callback: (feedback, bank) => {
+				if (self.taskData.hasOwnProperty(feedback.options.timeline)) {
+					if(feedback.options.icons) {
+	          if(self.taskData[feedback.options.timeline].status == 2) { // play
+	            return {
+								png64: self.feedbacksSettings.images.play,
+								pngalignment: "center:bottom"
+							}
+						} else if(self.taskData[feedback.options.timeline].status == 1) { //pause
+							return {
+								png64: self.feedbacksSettings.images.pause,
+								pngalignment: "center:bottom"
+							 }
+						} else if(self.taskData[feedback.options.timeline].status == 0) { //stop
+							return {
+								png64: self.feedbacksSettings.images.stop,
+								pngalignment: "center:bottom"
+							}
+						}
+					} else {
+						if(self.taskData[feedback.options.timeline].status == 2) { //play
+	            return {
+	              color   : feedback.options.playfg,
+	              bgcolor : feedback.options.playbg
+	            };
+	          } else if(self.taskData[feedback.options.timeline].status == 1) { //pause
+							return {
+								color   : feedback.options.pausefg,
+								bgcolor : feedback.options.pausebg
+							};
+						} else if(self.taskData[feedback.options.timeline].status == 0) { // stop
+							return {
+								color   : feedback.options.stopfg,
+								bgcolor : feedback.options.stopbg
+							};
+						}
+					}
+        }
+			},
+			subscribe: function (feedback) {
+				self.manageSubscription(feedback.options.timeline, true);
+	    },
+	    unsubscribe: function (feedback) {
+				self.manageSubscription(feedback.options.timeline, false);
+	    }
+		};
+
+		feedbacks['task_playing'] = {
```

Comment:
This should be re-implemented as a [boolean feedback](https://github.com/bitfocus/companion/wiki/Feedback).

Reply:
I'll will do this ASAP